packages/widget/src/devMode/global.css

```diff
@@ -2,3 +2,7 @@ body {
   margin: 0;
   min-height: 100vh;
 }
+
+[data-logo="skip-go"] {
+  display: none !important;
+}
```

Comment:
This doesn't actually apply to the widget anywhere besides in local dev, I'll be doing this on the go.cosmos.network side to hide these logos like in the design